tutorials/4_RunningREMINDandMAgPIE.md

```diff
@@ -64,7 +66,23 @@ See comments in the head section of the file. Most importantly you need to provi
 ### Configure the scenario_config_coupled.csv of your choice
 
 By default these are (A) `scenario_config_coupled_SSPSDP.csv` and (B) `scenario_config_SSPSDP.csv`. A provides some extra information
-for coupled runs (e.g. which run should be started). All other settings are taken from B. Thus every scenario in A must also be present in B.
+for coupled runs (e.g. which run should be started, specific MAgPIE configurations), with one scenario per row and settings on the columns. B is a normal `scenario_config` that defines REMIND settings, as explained in [3_RunningBundleOfRuns](https://github.com/remindmodel/remind/blob/develop/tutorials/4_RunningREMINDandMAgPIE.md). All other settings are taken from B. Thus every scenario in A must also be present in B.
+
+All the columns must be present in the `scenario_config_coupled.csv` file, but most of them can be left blank. The required ones are:
+   - `scenario`: The name of the scenario, must be unique and match the `scenario` column in REMIND's `scenario_config.csv`
```

Comment:
isn't it called `title`?

Reply:
I think the name of the column does not matter. But it must be the first column. This is because the first column is used as row names of the settings objects, see [here](https://github.com/remindmodel/remind/blob/fe9bf59480ad406c351e12b10eedbee17f237f41/start_bundle_coupled.R#L75) the argument `row.names=1`. And the rownames are used as titles of the scenarios

Reply:
Makes sense, but I suppose it's useful to list it as `title ` here anyway. Doing it.